samples/optimizely-audiences/index.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Optimizely audiences</title>
+
+  <link rel="stylesheet" href="https://contentful.github.io/ui-extensions-sdk/cf-extension.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
+
+  <style>
+    body {
+      margin: 0;
+      background: none;
+      height: 130px;
+    }
+
+    select {
+      width: 100%;
+    }
+
+    .optimizely-logo {
+      height: 3em;
+      background: url(https://a.mtstatic.com/@public/production/site_6806/1461019488-logo.svg) no-repeat;
+      background-position: top -1em left -0px;
+      background-size: auto 4em;
+    }
+
+    .select2-choices {
+      min-height: 150px;
+      max-height: 150px;
+      overflow-y: auto;
+    }
+
+    .cf-field-error,
+    .cf-form-field {
+      display: none;
+    }
+  </style>
+  <script src="https://contentful.github.io/ui-extensions-sdk/cf-extension-api.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+</head>
+
+<body>
+  <p class="loading">Loading ...</p>
+  <p class="cf-field-error"></p>
+
+  <div class="cf-form-field">
+    <div class="select-container"></div>
+    <div class="cf-form-hint">
+      Please select the target audience for this content.
+    </div>
+    <div class="optimizely-logo"></div>
+  </div>
+
+
+
+  <script>
+    var cfExt = window.contentfulExtension || window.contentfulWidget;
```

Comment:
Drop `contentfulWidget`, we shouldn't support prehistoric SDK versions.

It's used once, so just do `window.contentfulExtension.init(...` below.

Reply:
done